splash/auth/oauth_resources.py

```diff
@@ -0,0 +1,47 @@
+from bson.json_util import dumps
+import jsonschema
+import json
+import os
+from flask import current_app
+from flask import request
+from flask_restful import Resource
+from google.oauth2 import id_token
+from google.auth.transport import requests
+
+dirname = os.path.dirname(__file__)
+auth_schema_file = open(os.path.join(dirname, "auth_schema.json"))
+AUTH_SCHEMA = json.load(auth_schema_file)
+auth_schema_file.close()
+
+
+CLIENT_ID = os.environ.get("GOOGLE_CLIENT_ID") + ".apps.googleusercontent.com" #TODO: Temporary solution, we will need to have a local config file
+#TODO: Find out how to handle errors for this endpoint
+
+class OAuthResource(Resource):
+    def post(self):
+        #TODO: Verify the schema before blindly assigning this to a variable
+        validator = jsonschema.Draft7Validator(AUTH_SCHEMA)
```

Comment:
This is maybe something you copied from somewhere else that also needs to be changed but, nevertheless, setting up the validator should be done outside of the post request....no point do doing it for every request. Recommend defining an __init__ method to put it into.

Reply:
Ah yes. Good idea. I will ensure to move it to a init method.